_includes/selector.html

```diff
@@ -380,9 +380,11 @@
 
             // all possible values
             python_vers: ["3.9", "3.10", "3.11"],
+            python_vers_stable: ["3.9", "3.10", "3.11"],
+            python_vers_nightly: ["3.10", "3.11"],
             conda_cuda_vers: ["11", "12"],
             pip_cuda_vers: ["11.4 - 11.8", "12"],
-            docker_cuda_vers: ["11.8", "12.0", "12.2", "12.5"],
+            docker_cuda_vers: ["11.8", "12.0", "12.5"],
```

Comment:
12.5 images have been available since 24.08: https://github.com/rapidsai/docker/pull/689

12.2 images will be dropped in 24.10: https://github.com/rapidsai/docker/pull/696

Reply:
I'm proposing just unconditionally dropping 12.2 for simplicity and to nudge people towards newer images.

But if reviewers disagree, I'd be happy to introduce a similar structure to what I'm proposing for Python versions, that allows the CUDA versions to differ by release. I think that'd be generically useful, just didn't add it here to limit the size of this PR.